packages/ui-core/src/components/table/table-cell.component.tsx

```diff
@@ -60,7 +64,8 @@ const TableCell: FC<Props> = ({
 
     return {
       cellValue,
-      textAlignment: valueType === 'number' ? 'right' : 'left',
+      textAlignment:
```

Comment:
This logic should be moved at `@keen.io/charts` level IMO. The `<TableCell />` should not have knowledge about formatters.